HTML/Mystery Widget.html

```diff
@@ -0,0 +1,41 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>Mystery Widget — Keep  Talking and Nobody Explodes Module</title>
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+    <style>
+        table {
+            margin: 1em auto;
+        }
+    </style>
+</head>
+<body>
+    <div class="section">
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">Mystery Widget</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/Mystery Widget.svg" class="diagram">
+                <h2>On the Subject of Mystery Widget</h2>
+                <p class="flavour-text">Not a boss, D@M.</p>
+
+                <p>This module will hide one of the widgets on the edge of the bomb until it is solved.</p>
+
+                <p>To solve this module, solve the module on its display. The display may then change.</p>
+
+                <p>When the display starts flashing, hold the button across exactly three ticks of the bomb's timer. The module will then solve, and the widget will be revealed.</p>
+            
+                <p>Note: If the display is impossible to solve, you may activate the failswitch. To do this, hold the button across one tick of the timer. This will award a strike, then the display will change.</p>
```

Comment:
Change "This will award a strike" to "This will remove a fifth of the time from the bomb's timer" if you'd please. Thank you!